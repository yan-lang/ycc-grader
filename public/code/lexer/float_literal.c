3.145
3.0
9.81235642
132315.134
321566132.222
123.1000012
0.001324
0.1230012