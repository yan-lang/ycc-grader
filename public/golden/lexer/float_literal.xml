<tokens>
    <token>
        <index>0</index>
        <type>FLOAT_LIT</type>
        <source>float_literal.c</source>
        <lexer>Lexer</lexer>
        <value>3.145</value>
        <line>1</line>
        <column>1</column>
        <start>0</start>
        <stop>5</stop>
    </token>
    <token>
        <index>1</index>
        <type>FLOAT_LIT</type>
        <source>float_literal.c</source>
        <lexer>Lexer</lexer>
        <value>3.0</value>
        <line>2</line>
        <column>1</column>
        <start>6</start>
        <stop>9</stop>
    </token>
    <token>
        <index>2</index>
        <type>FLOAT_LIT</type>
        <source>float_literal.c</source>
        <lexer>Lexer</lexer>
        <value>9.812356</value>
        <line>3</line>
        <column>1</column>
        <start>10</start>
        <stop>20</stop>
    </token>
    <token>
        <index>3</index>
        <type>FLOAT_LIT</type>
        <source>float_literal.c</source>
        <lexer>Lexer</lexer>
        <value>132315.14</value>
        <line>4</line>
        <column>1</column>
        <start>21</start>
        <stop>31</stop>
    </token>
    <token>
        <index>4</index>
        <type>FLOAT_LIT</type>
        <source>float_literal.c</source>
        <lexer>Lexer</lexer>
        <value>3.21566144E8</value>
        <line>5</line>
        <column>1</column>
        <start>32</start>
        <stop>45</stop>
    </token>
    <token>
        <index>5</index>
        <type>FLOAT_LIT</type>
        <source>float_literal.c</source>
        <lexer>Lexer</lexer>
        <value>123.1</value>
        <line>6</line>
        <column>1</column>
        <start>46</start>
        <stop>57</stop>
    </token>
    <token>
        <index>6</index>
        <type>FLOAT_LIT</type>
        <source>float_literal.c</source>
        <lexer>Lexer</lexer>
        <value>0.001324</value>
        <line>7</line>
        <column>1</column>
        <start>58</start>
        <stop>66</stop>
    </token>
    <token>
        <index>7</index>
        <type>FLOAT_LIT</type>
        <source>float_literal.c</source>
        <lexer>Lexer</lexer>
        <value>0.1230012</value>
        <line>8</line>
        <column>1</column>
        <start>67</start>
        <stop>76</stop>
    </token>
    <token>
        <index>8</index>
        <type>EOF</type>
        <source>float_literal.c</source>
        <lexer>Lexer</lexer>
        <value>null</value>
        <line>8</line>
        <column>10</column>
        <start>76</start>
        <stop>76</stop>
    </token>
</tokens>